### PR TITLE
Include and serve netlify-cms see #2

### PR DESCRIPTION
**Pull request by mryellow**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+
+  <title>Netlify Jekyll CMS</title>
```
Comment (erquhart) [inline index.html:6]: Including "Jekyll" here could be confusing for some, this isn't really tied to a build tool.

**Comment by erquhart** [comment]
@mryellow awesome to see this example, we definitely need more of this so the CMS doesn't feel locked in.

**Comment by mryellow** [comment]
@erquhart The trick will be doing it in a way where you don't end up elsewhere (like this odd little repo) but instead are steered through the process via configuration and docs on the framework end, with netlify.com getting it's exposure (and defaults) along the way. 

Maybe pulling the oauth provider into a netlify build, rather than pulling netlify into the provider would make more sense.

**Comment by vencax** [comment]
Sorry to be late. So IMHO this is ONLY the provider. You can use it as 3rd party lib to make your own deployment and not to hardwire i-serve-cms-and-do-auth-as-well style of deployment. Others would prefer just to build NetlifyCMS and put it along their website living in a git repo, while leaving authentication backend somewhere else.
Does it make sense?

**Comment by vencax** [comment]
@mryellow yes, totally agree with adding this as another route into netlify express static file server (if there is one?  haven not found it in https://github.com/netlify/netlify-cms). if there is nothing like this, you can create another repo (eg. netlify-cms-standalone) and bring netlify-cms and this provider together.

**Comment by mryellow** [comment]
>  So IMHO this is ONLY the provider.

Yeah I can see that being the issue here, just went for making something I can use.

Maybe a layout something like this would work:

https://github.com/colinskow/superlogin/blob/master/lib/routes.js

Where routes are defined for the oAuth process, but then can be used in a larger express app with other routes like:

`app.use('/auth', superlogin.router);`

**Pull request closed without merging**